src/zope/dublincore/interfaces.py

```diff
@@ -208,56 +196,52 @@ def setQualifiedRights(qualified_rights):
         The argument must be a sequence of Rights `IDublinCoreElementItem`.
         """
 
+
 class IDCDescriptiveProperties(Interface):
-    """Basic descriptive meta-data properties
-    """
+    """Basic descriptive meta-data properties"""
 
     title = TextLine(
-        title = u'Title',
-        description =
-        u"The first unqualified Dublin Core 'Title' element value."
-        )
+        title=u'Title',
+        description=u"The first unqualified Dublin Core 'Title' element value."
+    )
 
     description = Text(
-        title = u'Description',
-        description =
-        u"The first unqualified Dublin Core 'Description' element value.",
-        )
+        title=u'Description',
+        description=(
+            u"The first unqualified Dublin Core 'Description' element value.")
+    )
+
 
 class IDCTimes(Interface):
-    """Time properties
-    """
+    """Time properties"""
 
     created = Datetime(
-        title = u'Creation Date',
-        description =
-        u"The date and time that an object is created. "
-        u"\nThis is normally set automatically."
-        )
+        title=u'Creation Date',
+        description=u"The date and time that an object is created."
+                    u"\nThis is normally set automatically."
+    )
```

Comment:
These look like simple flake-8-pacifying reformattings but they actually change the contents of the string literals.

It's probably fine.  I can't think of a reason why you'd need a space in front of the newline.  (Unless maybe the description is rendered in an HTML `title` attribute, and there are browsers that drop newlines instead of replacing them with spaces.  Experiments show that both browsers in 2020 don't do nonsense like that.)